.github/workflows/ci.yml

```diff
@@ -14,3 +14,20 @@ jobs:
     # Only run cron on the silverstripe account
     if: (github.event_name == 'schedule' && github.repository_owner == 'silverstripe') || (github.event_name != 'schedule')
     uses: silverstripe/gha-ci/.github/workflows/ci.yml@v1
+    with:
+      # set phpunit to false to prevent automatic generation of mysql phpunit jobs
+      phpunit: false
+      extra_jobs: |
+        - php: 8.1
+          db: sqlite3
+          composer_args: --prefer-lowest
+          phpunit_suite: framework
+          preserve_vendor_tests: true
```

Comment:
I've added the `preserve_vendor_tests` statement here to match an upstream change on the CI workflow.